xunit.runner.visualstudio.nuspec

```diff
@@ -41,5 +41,8 @@ Supported platforms:
     <file src="build\xunit.runner.visualstudio.wpa81.targets"                                   target="build\wpa81\xunit.runner.visualstudio.targets" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.dll" target="build\wpa81\" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.pri" target="build\wpa81\" />
+
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.deps.json" target="build\netstandard1.5\" />
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.dll" target="build\netstandard1.5\" />
```

Comment:
If this targets `netstandard1.5`, it's not really .NET Core, right? It's just "modern class libraries in .xproj", right? Would it be fairer to classify this as "netstandard support" rather than ".NET Core support"?

Or, should it actually be targetting `netcoreapp1.0` instead?